cmd/keymasterd/main.go

```diff
@@ -354,9 +412,34 @@ func (state *RuntimeState) checkAuth(w http.ResponseWriter, r *http.Request) (st
 		return "", AuthTypeNone, err
 
 	}
+	if (info.AuthType & requiredAuthType) == 0 {
+		state.writeFailureResponse(w, r, http.StatusUnauthorized, "")
+		err := errors.New("Insufficeint Auth Level")
```

Comment:
Also not clear.

Reply:
similar as above a call was made, but the authorization level was not eough given the current user cookies.